x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.ts

```diff
@@ -68,6 +68,8 @@ export const callEnterpriseSearchConfigAPI = async ({
     warnMismatchedVersions(data?.version?.number, log);
 
     return {
+      enterpriseSearchVersion: data?.version?.number,
+      kibanaVersion: kibanaPackageJson.version,
```

Comment:
Minor optional nit, feel free to ignore: `warnMismatchedVersions` above could be modified to take a `kibanaVersion`, so it doesn't have to know to look at `kibanaPackageJson.version` itself.